GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/TestHelpers/NewUiTestHelper.cs

```diff
@@ -209,6 +210,7 @@ public static IContainer BuildContainerIoC(params object[] dbObjects)
             
             //Register all controllers - this is required to ensure KeyFilter is resolved in constructors
             builder.RegisterType<AccountCreationController>().InstancePerLifetimeScope();
+            builder.RegisterType<AdminOrganisationScopeController>().InstancePerLifetimeScope();
```

Comment:
I've pushed a snazzy way of doing this that doesn't require us to add a new line for each controller